static/admin-widget-style.css

```diff
@@ -23,25 +55,32 @@ div.smaily div.section > h2 { /* Selects the immediate child element that is ins
     margin-bottom: 10px;
 }
 
-.smaily-layouts {
-    width: 90%;
+@media (min-width: 500px) {
+	.smaily-layouts  {
```

Comment:
Formatting: too many space before the starting curly bracket. Please review all CSS rules, there are more with the same issue.